.github/scripts/install_kafka.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+install_java() {
```

Comment:
Can be replaced with official [setup-java](https://github.com/actions/setup-java) action, and removed from this script.